### DISCUSSION TITLE
Search results page

### DISCUSSION THREAD

**Discussion by LobsterDM** [tailpress/tailpress#121] [Q&A]
Hi
Loving this theme but it seems to be missing a search results page. I'm working on a new site where I want to allow site users to search through the posts but I can't seem to get a search results page working.

Any ideas on what's needed to get it working?

**Comment by jeffreyvr** [comment]
Hi @LobsterDM,

You can add the [get_search_form](https://developer.wordpress.org/reference/functions/get_search_form/) somewhere in the template. Then you could make a custom `search.php` template file to display the results to your liking.

**Reply by LobsterDM** [reply]
Thank you for your help. Much appreciated
